src/react/account/details/properties/AccountKeys.tsx

```diff
@@ -83,25 +92,41 @@ function AccountKeys({ account }: Props) {
       {
         Header: 'Created On',
         accessor: 'created_at',
-        headerStyle: {
+        cellStyle: {
+          flex: '0.25',
           textAlign: 'right',
+          marginRight: '1rem',
         },
+        sortType: (
+          accessKeyRow1: Row<AccessKey>,
+          accessKeyRow2: Row<AccessKey>,
+        ) => {
+          const createdAt1 = new Date(accessKeyRow1.original.created_at);
+          const createdAt2 = new Date(accessKeyRow2.original.created_at);
 
-        Cell({ value: created_at }: { value: Date }) {
-          return (
-            <TextAligner alignment="right">
-              {formatShortDate(new Date(created_at))}
-            </TextAligner>
-          );
+          if (createdAt1.getTime() < createdAt2.getTime()) {
+            return 1;
+          }
+
+          if (createdAt1.getTime() > createdAt2.getTime()) {
+            return -1;
+          }
+
+          return 0;
         },
 
-        width: 80,
+        Cell({ value: created_at }: { value: Date }) {
+          return formatShortDate(new Date(created_at));
+        },
```

Comment:
Are you sure about this typing ? Because if `value/created_at` is already a `Date` what's the point of calling `new Date(created_at)`.

nit: why using snake case here for `created_at` ?

Reply:
Oh I see it is defined using snake case in the data passed to the table ...

Reply:
Well, apparently `value` is not a Date, it's string, because when I removed it some tests started failing :D

Reply:
The typing of `value` was wrong, it's a string and converting it to date is needed